include/asterisk/stasis_channels.h

```diff
@@ -758,6 +767,42 @@ int ast_channel_snapshot_connected_line_equal(
 	const struct ast_channel_snapshot *old_snapshot,
 	const struct ast_channel_snapshot *new_snapshot);
 
+
+/*!
+ * \since 23
+ * \brief Message published during an "ARI" transfer
+ */
+struct ast_ari_transfer_message {
+	/*! The channel receiving the transfer request */
+	struct ast_channel_snapshot *source;
+	/*! The bridge associated with the source channel */
+	struct ast_bridge_snapshot *source_bridge;
+	/*! The peer channel */
+	struct ast_channel_snapshot *source_peer;
+	/*! Referer identity */
+	char referred_by[4097];
```

Comment:
Where do these 4097 hardcoded values come from?

Reply:
At this point I am not entirely sure. I wanted to avoid dynamically allocating this string and found(?) that 4096 is a practical limit for the SIP header value and then added a byte for NUL. 

I guess I should just dynamically allocate the memory and manage it?

Reply:
It's dynamically allocated now.